packages/angular/cli/models/command-runner.ts

```diff
@@ -85,6 +85,8 @@ export async function runCommand(commandMap: CommandMap,
 
   if (!Cmd) {
     const commandsDistance = {} as { [name: string]: number };
+
+    commandName = args[0].replace(/-*/, '');
```

Comment:
Could you use the regex `/^-+/` instead? This would remove `-`s in the middle of the name.